app/main/views/audits.py

```diff
@@ -4,14 +4,16 @@
 from sqlalchemy.exc import IntegrityError
 from sqlalchemy.sql.expression import true, false
 from sqlalchemy.orm import class_mapper
-from ...utils import pagination_links, get_valid_page_or_1
 from .. import main
 from ... import db, models
 from dmapiclient.audit import AuditTypes
 from dmutils.config import convert_to_boolean
 from dmutils.formats import DATE_FORMAT
 from ...validation import is_valid_date, is_valid_acknowledged_state
-from ...utils import get_json_from_request, json_has_required_keys, validate_and_return_updater_request
+from ...utils import (
+    get_json_from_request, json_has_required_keys, validate_and_return_updater_request,
+    single_result_response, paginated_result_response, get_valid_page_or_1
+)
```

Comment:
Again, _please_ can we formalize on single import per line, trailing comma, preferably alphabetized.

Reply:
when i say import i obviously mean imported object

Reply:
lovely thankyou